convert_corpora.rb

```diff
@@ -0,0 +1,19 @@
+# encoding: UTF-8
```

Comment:
Better rewrite it on python and put to evaluate folder - all useful scripts are stored there for now.